src/Stryker.Core/Stryker.Core/Mutators/NullCoalescingExpressionMutator.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Stryker.Core.Mutants;
+using System.Collections.Generic;
+
+namespace Stryker.Core.Mutators
+{
+    public class NullCoalescingExpressionMutator : MutatorBase<BinaryExpressionSyntax>, IMutator
+    {
+        public override MutationLevel MutationLevel => MutationLevel.Basic;
+
+        public override IEnumerable<Mutation> ApplyMutations(BinaryExpressionSyntax node)
+        {
+            if (node.Kind() == SyntaxKind.CoalesceExpression)
+            {
+                var replacementNode = SyntaxFactory.BinaryExpression(SyntaxKind.CoalesceExpression, node.Right, node.Left); // Flip left and right
+                replacementNode = replacementNode.WithOperatorToken(replacementNode.OperatorToken.WithTriviaFrom(node.OperatorToken).WithLeadingTrivia(node.Left.GetTrailingTrivia()));
+                yield return new Mutation
+                {
+                    OriginalNode = node,
+                    ReplacementNode = replacementNode,
+                    DisplayName = $"Flip left and right hand side",
```

Comment:
All our mutation names are in the format of "... mutation (... to ...)". So in this case that would be "Null coalescing mutation (Left to right)". The other display names should be in the same format